Makefile

```diff
@@ -14,38 +14,23 @@
 # `clean` run clean on esptool2, rboot, and rboot-sampleproject with inherited
 #				 configuration
 
-ESPTOOL2      ?= $(abspath esptool2/esptool2.exe)
-SDK_BASE      ?= C:/esp_iot_sdk_v1.1.2
-XTENSA_BINDIR ?= C:/xtensa-lx106-elf/bin

```

Comment:
i'm not an expert with make, is it possible to detect if the compiler is in the path and if not use a variable to allow the location to be specified? i don;t have it in my path, so i like having it defined here